src/shims/unix/solarish/foreign_items.rs

```diff
@@ -69,6 +69,31 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 this.write_null(dest)?;
             }
 
+            "pset_info" if this.frame_in_std() => {
```

Comment:
```suggestion
            "pset_info" => {
```
Now that you implemented all the checks, this can be always enabled. :)